yarn-project/circuits.js/src/structs/kernel/__snapshots__/index.test.ts.snap

```diff
@@ -337,6 +337,8 @@ function_tree_root: 0x3
 contract_address_salt: 0x4
 portal_contract_address: 0x505050505050505050505050505050505050505
 
+chain_id: 0x2111
```

Comment:
why are these exactly the same?

Reply:
![Confused Thinking GIF](https://media0.giphy.com/media/WRQBXSCnEFJIuxktnw/200.gif)

WDYM? `2111 != 2211`